ppr-ui/src/components/mhrRegistration/HomeLocation/HomeLocationType.vue

```diff
@@ -346,15 +349,18 @@ export default defineComponent({
       localState.otherTypeOption = null
       localState.dealerManufacturerLot = ''
       localState.toggleInfoForm = false
+      setIsManualLocation(false)
+      localState.locationInfo = resetLocationInfoFields(localState.locationInfo)
```

Comment:
Is the argument required?
In this scenario you are mutating locationInfo vs creating/returning/assigning a new object. 
This isn't wrong but i'm not entirely sure of the purpose, since you've explicitly defined what properties you are mutating back to empty strings. 

This argument might applicable if `resetLocationInfoFields` was spreading or iterating over each property vs explicitly reseting specific values. 

I could be missing something though, please let me know if so!

Reply:
Hmm could we just initialize a new empty obj `localState.locationInfo = {} as MhrLocationInfoIF`

Reply:
That does work. That was my first approach ``localState.locationInfo = {}``, but I was missing the ``as MhrLocationInfoIF``. I'll make that change as it is a lot simpler